documentation/src/main/asciidoc/upgrading/upgrading-content.adoc

```diff
@@ -4,6 +4,11 @@
 
 The previously deprecated DistributedExecutor is now removed. References should be updated to use ClusterExecutor.
 
+=== Removed the Tree module
+
+TreeCache has been unsupported for a long time and was only intended as a quick stopgap for JBossCache users. The module has now been removed
+completely.
```

Comment:
Nitpicking: as a rule, we should split lines to on `.`, although in this case there won't be any updates so it won't make any diffs nicer.